is_core/locale/cs/LC_MESSAGES/django.po

```diff
@@ -122,11 +122,19 @@ msgstr "Služba je dočasně nedostupná"
 msgid "Readonly field can not be validated"
 msgstr "Pole pro čtení nemůže být validováno"
 
-#: forms/widgets.py:61 forms/widgets.py:67 forms/widgets.py:417
-#: forms/widgets.py:426
+#: forms/widgets.py:61 forms/widgets.py:67 forms/widgets.py:419
+#: forms/widgets.py:429
 msgid "Search..."
 msgstr "Hledat..."
 
+#: forms/widgets.py:420
+msgid "Insert primary key"
+msgstr "Vložte privátní klíč"
```

Comment:
I guess here should be "primární" not "privátní" - same applies lower.

Reply:
Yes thank you